src/minecraft/MinecraftManager.js

```diff
@@ -14,7 +13,7 @@ class MinecraftManager extends CommunicationBridge {
 
     this.stateHandler = new StateHandler(this)
     this.errorHandler = new ErrorHandler(this)
-    this.chatHandler = new ChatHandler(this, new CommandHandler(this))
+    this.chatHandler = new ChatHandler(this)
```

Comment:
I'm not sure why this was deleted? The command handler is required in the chat handler for the bot to function properly, without it you can't invoke any in-game commands through the bot.

Reply:
Didn't realize but I was using a modified version and older of the chat bridge, fixed it.